molecule/provisioner/ansible/playbooks/docker/create.yml

```diff
@@ -75,6 +75,7 @@
         networks: "{{ item.networks | default(omit) }}"
         network_mode: "{{ item.network_mode | default(omit) }}"
         dns_servers: "{{ item.dns_servers | default(omit) }}"
+        etc_hosts: "{{ item.etc_hosts | default(omit) }}"
```

Comment:
Just to mark with review interface. Please see https://github.com/ansible/molecule/pull/1571#issuecomment-458108002.